patches/native-base+2.12.1.patch

```diff
@@ -1,18 +1,41 @@
-patch-package
+diff --git a/node_modules/native-base/.DS_Store b/node_modules/native-base/.DS_Store
```

Comment:
This patch is not totally right
It tries to patch some file that exists only on your computer

Reply:
Honestly, I can't understand why. Now I create it manually.